src/docs/user/references/VariablesReference.adoc

```diff
@@ -1,140 +1,197 @@
 This section describes useful variables, their scope, and their usage.
 
-[cols="1,1,1,1", options="header"]
+The following script names refer to:
+
+* <<_selection,selection>> : a script executed to select nodes suitable to execute the task. This script is executed on many nodes from the scheduling loop, before the task is deployed.
+* <<_fork_environment, environment>> : the default mode to execute a task is to fork a dedicated JVM from the selected ProActive Node. The environment script, if defined in the workflow, is executed on the node to
+ let the user define which environment parameters will be given to the forked JVM.
+* <<_pre_post_clean, pre>> : a script which is executed _before_ the task execution.
+* <<_scripting_language_support,task>> : the main script of a task when the task is a *script task*.
+* <<_pre_post_clean, post>> : a script which is executed _after_ the task execution, if there was no error inside the task.
```

Comment:
Is it executed if tasked failed because of node failure? would be nice to explain the behaviour regarding error and failure which is different in our vocabulary (cf. names in Scheduler web interface).